compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -202,6 +215,8 @@ error.import.declaration.after.other.declarations=import declaration after other
 error.annotations.attached.to.expression=annotations attached to expression
 error.invalid.expression.in.start.action=invalid expression in start action
 error.same.object.type.qualifier=same object type qualifier
+error.duplicate.qualifier=duplicate qualifier
```

Comment:
`duplicate ''{0}'' qualifier` would be better

Reply:
thanx will change